### ISSUE TITLE
QF tab associated identity: overwrite of recipients by To address(es) even in reply mode 

### ISSUE THREAD

**Issue by macdeport** [RealRaven2000/QuickFolders#92]
![qf-advanced-properties-to-parameter](https://user-images.githubusercontent.com/26068595/103335877-b8165600-4a76-11eb-89d9-260b64a001e6.jpg)

**Comment by RealRaven2000** [comment]
Try this quick fix:
[QuickFolders-wx-5.1.1pre5.zip](https://github.com/RealRaven2000/QuickFolders/files/5754186/QuickFolders-wx-5.1.1pre5.zip)

It avoids setting the "to address" for the following case:
```
gMsgCompose.type == msgComposeType.Reply
gMsgCompose.type == msgComposeType.ReplyAll
gMsgCompose.type == msgComposeType.ReplyToSender
gMsgCompose.type == msgComposeType.ReplyToGroup
gMsgCompose.type == msgComposeType.ReplyToSenderAndGroup
gMsgCompose.type == msgComposeType.ReplyToList
```

**Comment by macdeport** [comment]
Issue #92 fixed 👍

**Comment by RealRaven2000** [comment]
Addressed in Release 5.2 (published on 22/01/2021)
